easybuild/easyconfigs/c/CPLEX/CPLEX-12.6.3.eb

```diff
@@ -0,0 +1,26 @@
+
+name = 'CPLEX'
+version = '12.6.3'
+
+homepage = 'http://www-01.ibm.com/software/commerce/optimization/cplex-optimizer/'
+
+description = """
+ IBM ILOG CPLEX Optimizer's mathematical programming technology enables
+ analytical decision support for improving efficiency, reducing costs,
+ and increasing profitability.
+"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# the Academic Initiative version (as used in this file) can be downloaded as described on
+# https://www.ibm.com/developerworks/community/blogs/jfp/entry/cplex_studio_in_ibm_academic_initiative?lang=en
+# a restricted "Community edition" version can be found on
+# https://www-01.ibm.com/software/websphere/products/optimization/cplex-studio-community-edition/
+sources = ['cplex_studio%s.linux-x86-64.bin' % ''.join(version.split('.'))]
+checksums = ['fe3b7e13c9e0d5a5025106488174ce1ef39db84827c00f123c92eb6fbf5df97a']
```

Comment:
Hmm, I have a different checksum, download is from June 10th 2016...

```
1aca03e070fd653c1a9a3d9b26ecb30fae8117982eb6110a5ac9f9235eeb0541  cplex_studio1263.linux-x86-64.bin
```

Maybe there's some kind of fingerprinting going on?